### DISCUSSION TITLE
Variants analysis in parent grain analysis

### DISCUSSION THREAD

**Discussion by KimMS32** [mtex-toolbox/mtex#2095] [Ask Anything]
Hi, 
I'm working on parent grain analysis for martensitic steels. While I try to follow the MTEX tutorial, I have a question about it.
in the tutorial, we can draw the variants map like below. in here, how to show the colorbar of variants? 
![MaParentGrainReconstruction_10](https://github.com/mtex-toolbox/mtex/assets/117698269/069e2397-bd8b-49ee-9244-19202750e7fe)

and also, is it possible that drawing the number fraction graph of variants?

**Comment by AzdiarGazder** [comment]
Hi @KimMS32,

Please have a look at:
https://github.com/ORTools4MTEX/ORTools
The functions within ORTools were created to help users easily produce publication-ready images.

Hope this helps.

Warm regards,
Azdi
